presto-main/src/test/java/io/prestosql/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -265,6 +269,32 @@ public void testFromUnixTimeWithTimeZone()
         assertFunction(format("from_unixtime(7200, '%s')", zoneId), TIMESTAMP_WITH_TIME_ZONE, toTimestampWithTimeZone(expected));
     }
 
+    @Test
+    public void testFromUnixTimeNanos_long()
+    {
+        LocalDateTime dateTime = LocalDateTime.of(2001, 1, 22, 3, 4, 5, 123456789);
```

Comment:
It seems that you can use `DateTimeTestingUtils.sqlTimestampOf` to construct expected value.
If not, please extract the method needed to construct expected value.
Preferably the test here would be just a series of:
```
assertFunction(from_unixtime_nanos(....), TIMESTAMP_NANOS, sqlTimestampOf(......)
assertFunction(from_unixtime_nanos(....), TIMESTAMP_NANOS, sqlTimestampOf(......)
```